tests/tests_app/cli/test_cli.py

```diff
@@ -61,6 +77,55 @@ def test_main_lightning_cli_help():
     assert "frontend" not in res
 
 
+@mock.patch("lightning_cloud.login.Auth.authenticate", MagicMock())
+@mock.patch("lightning_app.cli.cmd_clusters.AWSClusterManager.create")
+def test_clusters_create(create: mock.MagicMock):
+    runner = CliRunner()
+    runner.invoke(
+        create_cluster,
+        [
+            "test-7",
+            "--provider",
+            "aws",
+            "--external-id",
+            "dummy",
+            "--role-arn",
+            "arn:aws:iam::1234567890:role/lai-byoc",
+            "--instance-types",
+            "t2.small",
+        ],
+    )
+
+    create.assert_called_once_with(
+        cluster_name="test-7",
+        region="us-east-1",
+        role_arn="arn:aws:iam::1234567890:role/lai-byoc",
+        external_id="dummy",
+        instance_types=["t2.small"],
+        edit_before_creation=False,
+        cost_savings=False,
+        wait=False,
+    )
+
+
+@mock.patch("lightning_cloud.login.Auth.authenticate", MagicMock())
+@mock.patch("lightning_app.cli.cmd_clusters.AWSClusterManager.list")
+def test_clusters_list(list: mock.MagicMock):
```

Comment:
Might be worth patching LightningClient to make sure you receive the list.

Reply:
done in separate tests